From: Stuart Prescott <stuart@debian.org>
Date: Fri, 3 May 2024 16:48:49 +1000
Subject: Fix missing subparser in test harness

Not needed in the build but needed for debugging the test harness...
---
 testing/command.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/testing/command.py b/testing/command.py
index 31a48f8..859ebf1 100644
--- a/testing/command.py
+++ b/testing/command.py
@@ -194,6 +194,7 @@ def main():
         type=int,
         help="use build number n (0-based), latest = -1 (default)",
     )
+    parser_list = subparsers.add_parser("list")
     args = parser.parse_args()
 
     if hasattr(args, "buildno"):
